src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -22,34 +22,110 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
+
+{#
+# This form theme is an implementation of PrestaShop UI kit, it based on the bootstrap4 layout and overrides a few
+# blocks to match our expected integration of the UI kit.
+#
+# This theme uses/extends the basic bootstrap_4_layout meaning the form labels/inputs are displayed one under the other.
+# Unlike our prestashop_ui_kit.html.twig theme which extends this base kit but uses bootstrap_4_horizontal_layout as a
+# base, where labels/inputs are displayed horizontally in a column way.
+#}
+
+{#
+# We need to radio_widget and checkbox_widget from the original bootsrap 4 layout because we need to add form-check-radio
+# on the last wrapping div which is not possible with the attributes, so we basically copied the content, but then we cannot
+# use parent() to render the internal widget because it would reuse the one from bootstrap_4_layout and duplicate content.
+#
+# So we extract the initial widget from the bootstrap_base_layout to inject them as the internal widget, it is important
```

Comment:
Even if the explanations here are complete, I think it would be worth it to later write a short statement in the devdocs explaining
- each Form Theme purpose
- the relationships between available Form Themes

Reply:
Yep that's planned, I thougth a full doc of how to use the prestashop UI kit is needed
With examples for the new option we added via form extensions, the difference between the regular UI kit and the base one, along with screenshots and all

This comment is only here to avoid future developers to break anything, because the whole thing is quite sensitive ^^